components/prism-vhdl.js

```diff
@@ -0,0 +1,24 @@
+Prism.languages.vhdl = {
+	'comment': /(--[^\r\n]+)/,
+	// support for all logic vectors
+	'vhdl-vectors': {
+		'pattern': /(\b[oOxXbB]"[\dA-Fa-f_]+"|"[UuXx01ZzWwLlHh-]+")/,
```

Comment:
It looks like all letters are uppercase or lowercase: you could use the `i` modifier to simplify the regexp.